frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts

```diff
@@ -21,6 +23,12 @@ export const sharedVendorModules = [
  */
 export const overrideSharedModules = (entryModule: RemoteEntryModule) => {
   entryModule.override({
+    '@openshift-console/dynamic-plugin-sdk': async () => () =>
+      // eslint-disable-next-line
+      require('@console/dynamic-plugin-sdk/src/index-lib'),
+    '@openshift-console/dynamic-plugin-sdk/lib/*': async () => () =>
```

Comment:
Didn't know that we can use wildcards here, does it really work? :smile:

Reply:
eeh, not sure :) This is some leftover. I replaced it with full path to internal api